core/spring/src/main/java/org/apache/syncope/core/spring/security/AuthDataAccessor.java

```diff
@@ -108,35 +111,27 @@
 
     @Autowired
     protected AccessTokenDAO accessTokenDAO;
-
-    @Autowired
-    private ConfParamOps confParamOps;
-
     @Autowired
     protected ConnectorFactory connFactory;
-
     @Autowired
     protected AuditManager auditManager;
-
     @Autowired
     protected MappingManager mappingManager;
-
     @Autowired
     protected ImplementationLookup implementationLookup;
-
+    @Autowired
+    private ConfParamOps confParamOps;
     private Map<String, JWTSSOProvider> jwtSSOProviders;
 
     public JWTSSOProvider getJWTSSOProvider(final String issuer) {
         synchronized (this) {
             if (jwtSSOProviders == null) {
                 jwtSSOProviders = new HashMap<>();
 
-                implementationLookup.getJWTSSOProviderClasses().stream().
-                        map(clazz -> (JWTSSOProvider) ApplicationContextProvider.getBeanFactory().
-                        createBean(clazz, AbstractBeanDefinition.AUTOWIRE_BY_TYPE, true)).
-                        forEachOrdered(jwtSSOProvider -> {
-                            jwtSSOProviders.put(jwtSSOProvider.getIssuer(), jwtSSOProvider);
-                        });
+                implementationLookup.getJWTSSOProviderClasses().stream()
```

Comment:
Avoid format changes, please.

WRT this specific change, I know it's more a matter of taste, but generally we prefer ending lines with dot rather than starting lines with dot.